src/App.tsx

```diff
@@ -1,7 +1,16 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import OAuth from './oauthConfig';
 import { AuthConfiguration } from 'react-native-app-auth';
 import { RootProviders, RootStack, init } from '@lifeomic/react-native-sdk';
+import { PushNotificationsScreen } from './Screens/PushNotificationsScreen';
+
+const PushNotificationScreenProvider = ({
+  children,
+}: {
+  children?: React.ReactNode;
+}) => {
+  return <PushNotificationsScreen />;
+};
```

Comment:
I'm a bit confused about this. Maybe you could clarify:

* This isn't really a provider. There isn't any shared context, state, or functionality being "provided" to its children
* This takes props, but doesn't do anything with them. It is just rendering a component and preventing the `RootStack` from rendering which seems like a problem.